### PR TITLE
community/py-greenlet: upgrade to 0.4.15

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: community/py-greenlet/APKBUILD [hunk 1]
```diff
@@ -11,8 +11,7 @@ license="MIT"
 depends=""
 makedepends="python2-dev py-setuptools python3-dev"
 subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
-	ppc64le_clobbering.patch"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
```
[the remark this answers is not in the document]
Comment (J0WI) [reply community/py-greenlet/APKBUILD]: Oops, rebased.

**Comment by andypost** [comment]
@J0WI now you're messed up with commit message(

**Comment by J0WI** [comment]
Fixed

File: community/py-greenlet/APKBUILD [hunk 2]
```diff
@@ -11,8 +11,7 @@ license="MIT"
 depends=""
 makedepends="python2-dev py-setuptools python3-dev"
 subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
-	ppc64le_clobbering.patch"
```
Comment (andypost) [inline community/py-greenlet/APKBUILD]: @J0WI did you test ppc64le build? or it was fixed upstream?

Comment (J0WI) [reply community/py-greenlet/APKBUILD]: I think this has been addressed in https://github.com/python-greenlet/greenlet/commit/9c8a65fddcffe161f38679a0b3289de9b25e2bc6

**Comment by andypost** [comment]
works for me
```diff
diff --git community/py-greenlet/APKBUILD community/py-greenlet/APKBUILD
index ab81f18bff..b9693cebc7 100644
--- community/py-greenlet/APKBUILD
+++ community/py-greenlet/APKBUILD
@@ -18,8 +18,8 @@ builddir="$srcdir/$_pkgname-$pkgver"
 build() {
        cd "$builddir"
        export CFLAGS="-O0" CPPFLAGS="-O0" CXXFLAGS="-O0"
-       python2 setup.py build || return 1
-       python3 setup.py build || return 1
+       python2 setup.py build
+       python3 setup.py build
 }
 
 package() {
@@ -35,6 +35,12 @@ _py() {
        $python setup.py install --prefix=/usr --root="$subpkgdir"
 }
 
+check() {
+       cd "$builddir"
+       python2 setup.py test
+       python3 setup.py test
+}
+
 _py2() {
        _py python2
        replaces="$pkgname"
```

**Comment by J0WI** [comment]
Thanks, added.

**Comment by andypost** [comment]
Commited f44ddea6 709480b8
btw it fails on ppc64le http://build.alpinelinux.org/buildlogs/build-edge-ppc64le/community/py-greenlet/py-greenlet-0.4.15-r0.log

**Pull request closed without merging**